src/doc/book/functions.md

```diff
@@ -221,15 +221,15 @@ If you add a main function that calls `diverges()` and run it, you’ll get
 some output that looks like this:
 
 ```text
-thread ‘<main>’ panicked at ‘This function never returns!’, hello.rs:2
+thread ‘main’ panicked at ‘This function never returns!’, hello.rs:2
```

Comment:
Unrelated to this change specifically, but it looks like some smart quotes snuck into this line.